ogr/services/gitlab/project.py

```diff
@@ -364,7 +364,7 @@ def get_branches(self) -> List[str]:
     def get_file_content(self, path, ref="master") -> str:
         try:
             file = self.gitlab_repo.files.get(file_path=path, ref=ref)
-            return str(file.decode())
+            return file.decode().decode()
```

Comment:
Really two times?..;)

Reply:
Yeah! looks weird the first one is calling [this](https://python-gitlab.readthedocs.io/en/stable/api/gitlab.v4.html#gitlab.v4.objects.ProjectFile.decode).